backend/neofs/neofs.go

```diff
@@ -0,0 +1,988 @@
+	"github.com/rclone/rclone/fs/hash"
+	"github.com/rclone/rclone/fs/walk"
+	"github.com/rclone/rclone/lib/bucket"
+)
+
+func init() {
+	fs.Register(&fs.RegInfo{
+		Name:        "neofs",
+		Description: "NeoFS - distributed, decentralized object storage",
+		NewFs:       NewFs,
+		Options: []fs.Option{
+			{
+				Name:     "neofs_endpoint",
+				Help:     "Endpoints to connect to NeoFS node",
+				Required: true,
+				Examples: []fs.OptionExample{
+					{
+						Value: "grpcs://st1.t5.fs.neo.org:8082",
+						Help:  "One Testnet endpoint.",
+					},
+					{
+						Value: "grpcs://st1.storage.fs.neo.org:8082",
+						Help:  "One Mainnet endpoint.",
+					},
+					{
+						Value: "s01.neofs.devenv:8080",
+						Help:  "One devenv endpoint.",
+					},
+					{
+						Value: "s01.neofs.devenv:8080 s02.neofs.devenv:8080",
+						Help:  "Multiple endpoints to form pool.",
+					},
+					{
+						Value: "s01.neofs.devenv:8080,1 s02.neofs.devenv:8080,2",
+						Help:  "Multiple endpoints with priority (less value is higher priority). Until s01 is healthy all request will be send to it.",
+					},
+					{
+						Value: "s01.neofs.devenv:8080,1,1 s02.neofs.devenv:8080,2,1 s03.neofs.devenv:8080,2,9",
+						Help:  "Multiple endpoints with priority and weights. After s01 is unhealthy requests will be send to s02 and s03 in proportions 10% and 90% respectively.",
+					},
+				},
+			},
+			{
+				Name:    "neofs_dial_timeout",
+				Default: fs.Duration(10 * time.Second),
+				Help:    "NeoFS connection timeout",
+			},
+			{
+				Name:    "neofs_healthcheck_timeout",
+				Default: fs.Duration(10 * time.Second),
+				Help:    "NeoFS request timeout",
+			},
+			{
+				Name:    "neofs_rebalance_interval",
+				Default: fs.Duration(30 * time.Second),
+				Help:    "NeoFS rebalance connections interval",
+			},
+			{
+				Name:    "neofs_session_expiration_duration",
+				Default: 100,
+				Help:    "NeoFS session expiration duration",
+			},
+			{
+				Name: "rpc_endpoint",
+				Help: "Endpoint to connect to NeoFS morph rpc node to resolve container names",
+				Examples: []fs.OptionExample{
+					{
+						Value: "https://rpc1.morph.t5.fs.neo.org:51331",
+						Help:  "Testnet morph rpc.",
+					},
+					{
+						Value: "https://rpc1.morph.fs.neo.org:40341",
+						Help:  "Mainnet morph rpc.",
+					},
+					{
+						Value: "http://morph-chain.neofs.devenv:30333",
+						Help:  "Devenv morph rpc.",
+					},
+				},
+			},
+			{
+				Name:     "wallet",
+				Help:     "Path to wallet",
+				Required: true,
+			},
+			{
+				Name: "address",
+				Help: "Address of account",
+			},
+			{
+				Name: "password",
+				Help: "Password to decrypt wallet",
+			},
+			{
+				Name:    "placement_policy",
+				Default: "REP 3",
+				Help:    "Placement policy for new containers",
+				Examples: []fs.OptionExample{
+					{
+						Value: "REP 3",
+						Help:  "Container will have 3 replicas",
+					},
+				},
+			},
+			{
+				Name:    "basic_acl",
+				Default: "0x1fbf8cff",
```

Comment:
Maybe add the default value to the list of examples, to get documented what the default actually means?